src/styles/blocks/container.scss

```diff
@@ -0,0 +1,16 @@
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  box-sizing: border-box;
+  color: #060b35;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
```

Comment:
it is not quite correct to set the width of the container to 200 px, since it can contain several cards